test/Makefile.include

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2013-2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+EXTRA_DIST += test/functional test/util/bctest.py test/util/bitcoin-util-test.py test/util/data
+
+check-local:
+	@echo "Running test/util/bitcoin-util-test.py..."
+	$(PYTHON) $(top_builddir)/test/util/bitcoin-util-test.py
```

Comment:
I think all of these can just be (builddir), since this file is included rather than invoked recursively.